draft-ietf-httpbis-priority.md

```diff
@@ -541,12 +541,19 @@ background urgency being associated. However, in the worst case, the asymmetry
 between the precedence declared by multiple clients might cause responses going
 to one end client to be delayed totally after those going to another.
 
-In order to mitigate this fairness problem, when a server responds to a request
-that is known to have come through an intermediary, the server SHOULD prioritize
-the response as if it was assigned the priority of  `u=1, i`
-(i.e. round-robin) regardless of the value of the Priority header field being
-transmitted, unless the server knows the intermediary is not
-coalescing requests from multiple clients.
+In order to mitigate this fairness problem, a server could use knowledge about
+the intermediary as another signal in its prioritization decisions. For
+instance, if a server knows the intermediary is coalescing requests, then
+it could prioritize responses using a fixed urgency level with incremental
+loading. Treating all requests from an intermediary in this way would result in
```

Comment:
I don't know why you would recommend incremental loading at all.  You might just say "assign all requests the same priority".  And then add the note about where the bottleneck is.

Reply:
In this context, incremental means round-robin. We don't care how the thing receiving the response processes it. But we do want to avoid the server responding as FIFO, which might cause a different type of fairness problem.

Reply:
Knowing that the intent of this PR is to be less specific, I think we can get rid of suggestions that refer to the specification (i.e., use of words like "urgency" and "incremental").

Rather, we can condense this sentence and the following one to something like: _For instance, if a server knows that the intermediary is coalescing requests, it could serve the responses in round-robin_.

Reply:
agreed, thanks!